src/components/FeedbackCard.tsx

```diff
@@ -67,6 +71,7 @@ const FeedbackCard = ({ prompt, isArticle, ...props }: FeedbackCardProps) => {
       mt="8"
       w="full"
       {...props}
+      dir={dir}
```

Comment:
This is an exception where the `dir` prop should be kept, as flexbox is controlling the content positioning here.

Reply:
Honestly I'm not having any problems with flex boxes here...

I'll double check all of your comments, but the reason for these was not _just_ to use the `dir` for the chosen locale, but to specifically do so at the component level for components that:

1. Exclusively use the "common" namespace for translations (which _every_ language has before even being listed
2. Are (or potentially) nested inside _page content_ which may end up NOT being translated.

In this case (ie. [/ar/developers/docs/blocks](https://ethereum.org/ar/developers/docs/blocks)) `html` could be set to `dir="rtl"`, the _main content div_ could be set with `dir="ltr"` (overriden because `contentNotTranslated=true`), but then some components (ie. `FeedbackCard`) are fully translated, which should again run **rtl**. 

The change here addressed this by independently setting the `dir` based on locale, without relying on `html[dir]` since the content it's nested in may flip it back.

Reply:
Got it. Agreed, I think we need this for these "global" components.